README.md

```diff
@@ -170,6 +171,21 @@ genres:
 { keepZeroFacets : true } // default: false
 ```
 
+### Matching strategy
+
+`matchingStrategy` gives you the possibility to chose how Meilisearch should handle the presence of multiple query words.
+
+For example, if your query is `Hello world` by default Meilisearch returns documents containing either both `Hello` and `world` or documents that only contain `hello`. This is the `last` strategy, where words are stripped from the right.
+The other strategy is `all`, where both `hello` and `worlds` **must** be present in a document for it to be returned.
+
+// TODO: add documentation link
```

Comment:
You might want to add it 😇

Reply:
Should have added it as a todo in the body of this PR! Thanks for noticing